CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## v18.5.0.0
+* Upgrade to [Cumulus v18.5.0](https://github.com/nasa/cumulus/releases/tag/v18.5.0)
+
```

Comment:
I think we should add a note that says that there are additional instructions to upgrade the RDS. Similar to the changelog for `v18.2.0.0`